package.json

```diff
@@ -53,7 +53,7 @@
   "main": "lib/index.js",
   "name": "@open-turo/semantic-release-config",
   "peerDependencies": {
-    "semantic-release": ">=21.1.1"
+    "semantic-release": ">=22"
```

Comment:
Do we really need to enforce 22+ for this? The config file hasn't changed and semantic-release is just a dev dependency here.

Reply:
We have warning because of the gradle-semantic-release plugin I believe and I think there are other internal libs that require 22+

Reply:
Confirmed here https://github.com/KengoTODA/gradle-semantic-release-plugin/blob/master/package.json#L53

<img width="437" alt="image" src="https://github.com/open-turo/semantic-release-config/assets/1235917/afa96078-c4cb-4edb-8211-f09ddd1b604f">